volatility3/framework/plugins/layerwriter.py

```diff
@@ -95,15 +100,16 @@ def _generator(self):
             if not self.config["layers"]:
                 self.config["layers"] = []
                 for name in self.context.layers:
-                    if not self.context.layers[name].metadata.get("mapped", False):
+                    if "mapped" not in self.context.layers[name].metadata:
                         self.config["layers"] = [name]
 
             for name in self.config["layers"]:
                 # Check the layer exists and validate the output file
                 if name not in self.context.layers:
                     yield 0, (f"Layer Name {name} does not exist",)
                 else:
-                    output_name = self.config.get("output", ".".join([name, "raw"]))
+                    default_output_name = f"{name}.raw"
```

Comment:
Happy to do this for clarity, just wondering if there's any other reason for it?

Reply:
Well again, just to keep things clear and avoid any extra code. 
IMO instantiate a list to then join the two elements with a dot and instantiate the final string is a bit excessive for such a simple task. Anyway, the main reason is clarity  `f"{name}.raw"` vs `".".join([name, "raw"])`